packages/server/graphql/public/mutations/updateTemplateCategory.ts

```diff
@@ -0,0 +1,38 @@
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import getKysely from '../../../postgres/getKysely'
+import publish from '../../../utils/publish'
+import {MutationResolvers} from '../resolverTypes'
+
+const updateTemplateCategory: MutationResolvers['updateTemplateCategory'] = async (
+  _source,
+  {templateId, mainCategory},
+  {dataLoader, socketId: mutatorId}
+) => {
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+  const pg = getKysely()
+
+  // VALIDATION
+  if (mainCategory.length < 1) return {error: {message: 'Category name cannot be blank'}}
+  if (mainCategory.length > 100) return {error: {message: 'Category name is too long'}}
+
+  // I'd like to relax this later, hence the separate validation from string length
+  const stockCategories = ['retrospective', 'estimation', 'standup', 'feedback', 'strategy']
```

Comment:
+1 extract to constant array, i.e. `STOCK_CATEGORIES` outside of the resolver?

+1 `CATEGORY_OPTIONS`?

Reply:
will do